Rota-Process.md

```diff
@@ -0,0 +1,302 @@
+
+  - `crash`
+  - `data-loss`
+  - `hang`
+  - `resource-leak`
+  - `unreliable`
+
+  If you add one of these labels, please also
+  [contact the team](https://github.com/kata-containers/community#join-us)
+  so they are aware of the issue as soon as possible.
+
+- Top priority
+
+  If an issue is extremely urgent and should be fixed as soon as possible, you
+  should add the `highest-priority` label to flag this to the entire team.
+
+- Release gating
+
+  If you think the issue should be fixed before the next release, add the
+  `release-gating` label.
+
+- Important medium / long term task
+
+  If an issue is an very important but is not time-critical, you should add
+  the `highest-severity` labels.
+
+- Simple task for new contributors
+
+  If you think the issue is a simple, relatively quick to resolve and
+  "self-contained", you should add the `good-first-issue` issue to mark it as
+  being suitable for a new contributor to look at.
+
+- Invalid
+
+  If you think the issue is somehow not appropriate, add the `invalid` label.
+
+- Duplicate
+
+  If the issue is a duplicate of an existing issue you've already seen, add
+  the `duplicate` label along with a comment referencing the original.
+
+- Limitation
+
+  If the issue describes a system limitation, add the `limitation` label.
+
+  If the issue is a
+  [known limitation](https://github.com/kata-containers/documentation/blob/master/Limitations.md),
+  close the issue with a comment referencing the original limitation issue
+  number so the issue author understands why the issue was closed.
+
+- Teams
+
+  If you know which team the issue should be assigned to, add one of the
+  `team/*` labels.
+
+- Related project
+
+  If the issue relates to another project, add one of the `related/*`
+  labels.
+
+- Issue is lacking something
+
+  If you think the issue is incomplete somehow, consider adding one of the
+  `needs-*` labels.
+
+##### General process
+
+BUG: FIXME: link to issue reports in wiki.
+
+We use
+[GitHub issue templates](https://github.com/kata-containers/.github/tree/master/.github/ISSUE_TEMPLATE)
+to help guide users to raising the correct type of issue. All these templates
+also automatically add a `needs-review` label to the issues.
+
+The idea of this label is to highlight issues that have *not* yet been
+reviewed by the team.
+
+Therefore, after you have reviewed an issue with this label (which will
+potentially have resulted in you applying additional labels), you should
+*remove* the `needs-review` label to signal that the issue has been reviewed.
+
+### Test stability
+
+The more PRs you look at, the more likely you are to notice patterns or
+anomalies of CI test failures.
+[Raising issues in the `tests` repository](https://github.com/kata-containers/tests/issues/new)
+helps us to identify "flaky" tests and prioritize the problematic bugs that
+make review/merges slower.
+
+Before raising an issue, search all issues (open and closed) looking for the
```

Comment:
I don't know if we can hotlink 'search all repos' off to a github link to the project wide search box can we? that'd be nice

Reply:
Good idea - done.